### PR TITLE
Prevent build chain debug logging from getting cut off.

### PR DESCRIPTION
**Pull request by apwilson**

### CODE REVIEW

File: sky/packages/sky/lib/src/widgets/framework.dart [hunk 1]
```diff
@@ -820,7 +820,7 @@ abstract class Component extends Widget {
       _debugIsBuilding = true;
       _debugPreviousComponent = _debugLastComponent;
       if (_debugLastComponent != null)
-        _debugLastComponent = "$_debugPreviousComponent -> ${this.toStringName()}";
+        _debugLastComponent = "$_debugPreviousComponent \n ${this.toStringName()}";
```
Comment (abarth) [inline sky/packages/sky/lib/src/widgets/framework.dart]: I'd remove the spaces around the \n

**Comment by abarth** [comment]
LGTM

**Comment by Hixie** [comment]
I'll take this one and make the debugging better here in general. Is there anything in particular you'd like to see that would help debugging?

**Comment by abarth** [comment]
I think this PR has been superseded by #1139

**Pull request closed without merging**